core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableInterpretable.java

```diff
@@ -126,7 +126,8 @@ public static Bindable toBindable(Map<String, Object> parameters,
       }
     } catch (Exception e) {
       throw Helper.INSTANCE.wrap("Error while compiling generated Java code:\n"
-          + s, e);
```

Comment:
This is unrelated changes about this ISSUE, If you think this message needs to improve, Please log a new issue in Jira.